core/modules/loaders.rs

```diff
@@ -179,29 +179,6 @@ impl ModuleLoader for ExtModuleLoader {
   }
 }
 
-impl Drop for ExtModuleLoader {
```

Comment:
What was the reason we deleted this code again? I couldn't find the thread on the old PR.

Reply:
https://github.com/denoland/deno/pull/19463#discussion_r1228740345